engine/src/main/java/org/hibernate/search/cfg/spi/impl/ExtendedSearchConfiguration.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.cfg.spi.impl;
```

Comment:
@Sanne @hferentschik : should this stay in this package or maybe be moved to org.hibernate.search.cfg.spi?